app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -2,6 +2,12 @@
     <LoadingIndicator data-test-loading-indicator @dark={{true}} />
 {{else}}
     <div local-class='filter-container'>
+        <div local-class='total-users'>
+            <span local-class='total-users-count'>
+                {{@totalUsers}}
+            </span>
+            <label>{{t 'institutions.dashboard.users_list.total_users'}}</label>
```

Comment:
`<label>` feels a bit inappropriate to use here, as there is no `<input>` that this is labelling. Could likely just be another `span` I think